### PR TITLE
Add Tony's Rochester File to Master

### LINKED ISSUE #11: Create a file for tonytraining rochester, ny
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a Pull Request
- [ ] Collaborate
- [ ] Receive Approval :+1: 
- [ ]  Merge :shipit:

### PR DESCRIPTION
**Pull request by tonytraining**
closes #11

### CODE REVIEW

File: tonytraining-rochester.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Rochester, NY
+
+#### Places To Go
+- Wegmans
+
+#### Places To Eat
+- Pontillo's Pizza
+
+#### Places To Avoid
+
```
Comment (cshing1) [inline tonytraining-rochester.md:12]: Do you know any places to avoid?

**Comment by cshing1** [comment]
:+1:

**Comment by tonytraining** [comment]
@khusban Dave check this out

**Comment by khusban** [comment]
@chrisatkins look good?

**Comment by khusban** [comment]
@tonytraining well?

**Pull request merged**